cypress/e2e/shared/dashboardsIndex.test.ts

```diff
@@ -761,4 +762,147 @@ describe('Dashboards', () => {
       })
     })
   })
+  it('changes time range', () => {
+    const dashName = 'dashboard'
+    cy.writeLPData({
+      lines: genCurve({points: 87000, freq: 10}),
+      offset: `${30 * 24 * 60}m`,
+      stagger: `${(30 * 24 * 60) / 87000}m`,
+    })
```

Comment:
the following lines should be within a `.then` to handle this cypress async task correctly

Reply:
actually just add `.should('equal', 'success')` to this line and I think you're good